js/libs/form.js

```diff
@@ -211,43 +211,13 @@ Fliplet.Widget.instance('form-builder', function(data) {
               }
               break;
 
-            case 'flCheckbox':
-              if (fieldData.length > 0) {
-                var inOptions = [];
-
-                fieldData.forEach(function (element) {
-                  var match = _.find(field.options, function (option) {
```

Comment:
@tonytlwu This code cannot be executed because the array field.options is empty unless it has been changed in the widget settings. So I decided to run this code in the checkbox component, because it always has relevant options.

Reply:
@OleksandrLavrynenko In that case, can you leave the following code so that developers can follow the thread in the future:

```js
case 'flCheckbox':
case 'flRadio':
  // Add comments to explain where to find the code for setting the field value and why it needs to be elsewhere
  break;
```